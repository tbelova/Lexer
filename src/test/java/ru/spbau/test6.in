if (a >= 5) then { 
	return 0; 
}
