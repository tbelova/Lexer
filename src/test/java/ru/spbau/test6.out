PROGRAM((1,0)(4,4))
   STATEMENTS((1,0)(3,0))
      STATEMENT((1,0)(3,0))
         IF_STATEMENT((1,0)(3,0))
            BOOLEAN_EXPRESSION((1,4)(1,9))
               GE((1,4)(1,9))
                  EXPRESSION((1,4)(1,4))
                     IDENT((1,4)(1,4))[a]
                  EXPRESSION((1,9)(1,9))
                     INTEGER((1,9)(1,9))[5]
            THEN_STATEMENT((1,17)(3,0))
               BLOCK((1,17)(3,0))
                  STATEMENTS((2,1)(2,9))
                     STATEMENT((2,1)(2,9))
                        RETURN((2,1)(2,8))
                           EXPRESSION((2,8)(2,8))
                              INTEGER((2,8)(2,8))[0]
